workers/unity/Packages/com.improbable.gdk.mobile/Android/Utility/AndroidDeviceInfo.cs

```diff
@@ -2,22 +2,22 @@
 
 namespace Improbable.Gdk.Mobile.Android
 {
+    public enum AndroidDeviceType
```

Comment:
Does the iOS version also return a device type?

Wondering if we can have a MobileDeviceType {Unknown, Physical, Virtual} in that case?

Reply:
yes, that would be perfect :)

Reply:
That would be nice and would enforce consistency between the two classes (Android and iOS) at compile time.

Reply:
Good point, let me add that

Reply:
Actually, after trying that - I am not sure that is a great idea: Android and iOS device types are different. Android uses emulator, while iOS uses simulator. Later on we might want to add more concrete device types, for instance separate them by generation or by different emulator types and having them all in one enum wouldn't make a lot of sense.
I am also not sure if the consistency here would be that beneficial.
What do people think?

Reply:
I would still vote for Firtina's suggestion for now: `MobileDeviceType {Unknown, Physical, Virtual} `

I agree that this might change later. and we will most likely want to differentiate between phone and tablet very soon.
However, for now I would keep it simple and only split it out when it becomes necessary